lib/output.js

```diff
@@ -622,7 +622,8 @@ function OutputStream(options) {
         }
 
         if (current_pos == 0) {
-            if (comments.length > 0 && options.shebang && comments[0].type == "comment5") {
+            if (comments.length > 0 && options.shebang && comments[0].type == "comment5"
```

Comment:
May be strict equality?
`comments[0].type === "comment5"`

Reply:
I think most old code is using "`.type == ""`"...

Reply:
Yes, it is. We can change this step-by-step. I mean not only `==` vs `===` but the whole code, like replacing `Object.create(null)` with `new Map()` or `new Set()`